src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Parser.cs

```diff
@@ -459,9 +461,13 @@ public IReadOnlyList<LoggerClass> GetLogClasses(IEnumerable<ClassDeclarationSynt
                         case "Message":
                             message = sm.GetConstantValue(a.Expression, _cancellationToken).ToString();
                             break;
+                        case "SkipEnabledCheck":
+                            var flag = (bool?)sm.GetConstantValue(a.Expression, _cancellationToken).Value;
```

Comment:
```suggestion
                            bool? flag = (bool?)sm.GetConstantValue(a.Expression, _cancellationToken).Value;
```
? (I don't know the exact policy here becuase of the cast).